.github/instructions/typescript-types.instructions.md

```
@@ -114,64 +114,21 @@ export interface CurrentUserRead extends UserRead, Permissions {
-## Integration with React Query
-```typescript
-// API query result types
-export type PatientQueryResult = {
-  data: PatientRead;
-  isLoading: boolean;
-  error: ApiError | null;
-  refetch: () => void;
-};
-```
\ No newline at end of file
+## Medical Data Privacy
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)